app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,44 @@
+module SessionsHelper
+  # frozen_string_literal: true
+  def sign_in(user)
+    session[:user_id] = user.id
+    current_user
+  end
```

Comment:
Hey, please not that code put in helper are code to be used in the views. take a look at this [where to put code](https://codefol.io/posts/where-do-i-put-my-code-in-rails-updated/)